### PR TITLE
Add route hint to copy button on person's profile if available

### LINKED ISSUE #788: Add "route hint" to copy button on a person's profile
The route hint isn't included when I click "copy" on my profile. The routehint should also be included.
https://github.com/stakwork/sphinx-tribes/assets/32662508/4dea98f0-6897-4015-ad08-d5ce0bea3636

### PR DESCRIPTION
**Pull request by Marvel-Ib**
## Describe your changes
This change add route hint if available to a user copied connection link.

in the format 

{public_key}:{route_hint}
## Issue ticket number and link
Closes #788 
## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] I have tested on Chrome and Firefox
- [ ] I have tested on a mobile device
- [ ] It has been deployed to people-test.sphinx.chat and tested by others

### CODE REVIEW

**Comment by Marvel-Ib** [comment]
@kevkevinpal , @ecurrencyhodler

**Review by kevkevinpal — APPROVED** [review]
lgtm @elraphty can you review and then merge if this looks good to you?

**Comment by elraphty** [comment]
@Marvel-Ib the route hint is also supposed to be added to the copy button, not just the QR Code.

**Comment by kevkevinpal** [comment]
also we need to add routehint to the copy button aswell

File: frontend/app/src/people/utils/ConnectCard.tsx [hunk 1]
```diff
@@ -95,6 +95,8 @@ export default function ConnectCard(props: ConnectCardProps) {
   const showAssignModal = () => setAssignModal(true);
 
   const qrString = person && person?.owner_pubkey ? makeConnectQR(person?.owner_pubkey) : '';
+  const ownerPubkey = person && person?.owner_pubkey ? `${person.owner_pubkey}` : ''
+  const routeHint = person && person?.route_hint ? `:${person.route_hint}` : ''
```
Comment (Marvel-Ib) [inline frontend/app/src/people/utils/ConnectCard.tsx:99]: hi @kevkevinpal  the ':' addition was actually handled here if route_hint is available

**Comment by Marvel-Ib** [comment]
> @Marvel-Ib the route hint is also supposed to be added to the copy button, not just the QR Code.

@elraphty  adding it to just the Qr code does the trick. if i press the copy button it actually copies it . tested and working on my end 

sorry about the late turnaround time. work as always

**Comment by Marvel-Ib** [comment]
oh yeah, @elraphty  any chance I might get help with the sphinx tribe go backend setup? I will love to contribute more to the backend of things

**Comment by ecurrencyhodler** [comment]
@elraphty can you take one more look at this pr to see if we can merge?

**Comment by elraphty** [comment]
@Marvel-Ib can you look into this extensively, I tested as both a Relay user and an LNUTH user, and both didn't work, user ```owner_route_hint``` instead of route_hint, this should fix this, thanks. sorry for the late response.

**Comment by Marvel-Ib** [comment]
oh i see. I will check it and close it then 

Thanks @elraphty

**Comment by Marvel-Ib** [comment]
@elraphty  it worked like a charm. I will clean up and make an updated push soon. Thanks

**Comment by elraphty** [comment]
> @elraphty it worked like a charm. I will clean up and make an updated push soon. Thanks

you are welcome @Marvel-Ib

**Comment by Marvel-Ib** [comment]
@elraphty  updated

**Comment by elraphty** [comment]
@Marvel-Ib please fix the prettier issues by running ```yarn run prettier``` on frontend/app folder

**Comment by Marvel-Ib** [comment]
@elraphty  done

**Comment by elraphty** [comment]
> @elraphty done

thanks

**Comment by elraphty** [comment]
@Marvel-Ib  please add the functionality to the copy button on the about page and push About Page, you only added the route hint to the connect QR Code copy, please quickly make this change so I can merge today, thanks.

**Pull request merged**